spec/acceptance/nodesets/ubuntu-1404-x64-vcloud.yml

```diff
@@ -0,0 +1,16 @@
+HOSTS:
```

Comment:
I think we should probably keep these in the Puppet Labs fork, along with other vendor modifications.